### ISSUE TITLE
not being able to access the "dodge" in the combat chat.

### ISSUE THREAD

**Issue by heliobolseiro** [Miskatonic-Investigative-Society/CoC7-FoundryVTT#512]
hello, my players in the chat, at the time of the fight, are not being able to access the "dodge", "counterattack", etc ... He gives the following warning: "user player x lacks permission to update ChatMessage viMyJfaY6WzrLJBg (this code varies) What could be happening?

**Comment by kirtanpl** [comment]
+1, had the same issue on my last play. Additionally, GM can select option for the player but when player rolls, message is empty (rolling works for GM)

**Comment by HavlockV** [comment]
Thanks for the feedback, it looks like a permission problem.
Could you send a screenshot of the chat log to help investigate this issue.
Also make sure that : 
- Your player owns the actor
- This actor is your player's character ( When your player will connect for the first time he will be asked to select a character amongst actors he owns. You can configure that as a GM by clicking the player's actor selection on the bottom left corner of the screen )
- When you created that actor's token ensure that the token data is linked to that actor (Token configuration, make sure the link actor's data box is ticked)

![image](https://user-images.githubusercontent.com/65915923/118399022-6d7b2c00-b65b-11eb-954e-fca7087d6d6c.png)

**Comment by kirtanpl** [comment]
We've debugged this with my players and apparently players can't fight each other. When one player selects other player's token and fights, targeted person can't fight back.
Checked all your three points and everything seems to be properly set.

![obraz_2021-05-16_182208](https://user-images.githubusercontent.com/1628319/118404533-9efff180-b673-11eb-8721-89af17002fc6.png)

**Comment by heliobolseiro** [comment]
![IMG-20210516-WA0021](https://user-images.githubusercontent.com/84214182/118405350-ea070e00-b64d-11eb-8ca5-4a087776fc4d.jpg)

**Comment by heliobolseiro** [comment]
Error

**Comment by HavlockV** [comment]
Can you show me your player's 3 config from GM interface (bottom left box as shown in the pic I've sent )

**Comment by heliobolseiro** [comment]
Havlock, I did everything you recommended and follow the images. What I noticed now is that when the combat is with NPCs, this bug does not happen. As I recently bought the foundry, I was testing the fighting with friends and it was always between players. Now, I made a test using an NPC I created and the player was able to activate the "dodge" or "counterattack" or "maneuver". It seems that the problem is between "players".
I know that fighting between players is not common in Call of Cthulhu, but it would be interesting to fix this supposed bug. Your system is very good and is to be congratulated. I will continue my campaign of Masks of Nyarlathotep in the foundry, by its system of rules. Before, I used fantasy grounds. I'm really enjoying the program and its rules.
and I will always be giving suggestions and accusing new things, all to help you improve the game.

![WhatsApp Image 2021-05-16 at 22 06 47](https://user-images.githubusercontent.com/84214182/118419922-d2eb0f00-b693-11eb-98e4-e583474bb0ea.jpeg)
![WhatsApp Image 2021-05-16 at 22 04 40](https://user-images.githubusercontent.com/84214182/118419928-d41c3c00-b693-11eb-868b-47d3342b435d.jpeg)
![WhatsApp Image 2021-05-16 at 22 03 03](https://user-images.githubusercontent.com/84214182/118419929-d41c3c00-b693-11eb-8818-275ed9a2e904.jpeg)
